core/src/main/java/org/apache/iceberg/GenericManifestFile.java

```diff
@@ -137,6 +138,26 @@ public GenericManifestFile(String path, long length, int specId, Long snapshotId
     this.fromProjectionPos = null;
   }
 
+  public GenericManifestFile(String path, long length, int specId, Long snapshotId, Long sequenceNumber,
```

Comment:
This is an internal class, so it shouldn't be necessary to add a new constructor.

Reply:
Agreed, no place uses it actually.